### PR TITLE
DOC-2642: Update Kafka Connector Quickstart with info about RBAC auth

### PR DESCRIPTION
**Pull request by dnault**
Document the 'connection.username' and 'connection.password' config
properties.

Plus a little bit of cleanup.

### CODE REVIEW

File: content/connectors/kafka-3.1/quickstart.dita [hunk 1]
```diff
@@ -68,10 +78,10 @@ use_snapshots=false
 {"event":"mutation","partition":104,"key":"landmark_5574","cas":1479128707104768,"bySeqno":3,"revSeqno":1,"expiration":{...
 ...
 ]]></codeblock>
-            <p>The output will display about 31,000 lines, each containing mutation (assuming sample bucket hasn't been modified after creation). 
-                The shape of the payload is controlled by the <codeph>dcp.message.converter.class</codeph> property of the connector config. 
-                By default it set to <codeph>com.couchbase.connect.kafka.converter.SchemaConverter</codeph>, which formats events into structure 
-                with meta information on the top level and optional value as bytestring in <codeph>content</codeph> property. 
+            <p>The output will display about 31,000 lines, each containing mutation (assuming sample bucket hasn't been modified after creation).
+                The shape of the payload is controlled by the <codeph>dcp.message.converter.class</codeph> property of the connector config.
+                By default it set to <codeph>com.couchbase.connect.kafka.converter.SchemaConverter</codeph>, which formats events into structure
```
Comment (ingenthr) [inline content/connectors/kafka-3.1/quickstart.dita]: "By default it **is** set to"

**Comment by dnault** [comment]
Thanks @ingenthr . Incorporated the suggested edits.

**Comment by dnault** [comment]
@avsej @ingenthr should we hold off on merging this until after the next Kafka Connector release? I'm concerned that the version of the connector in the wild does not support the `connection.username` property, and folks may have trouble using it with a 5.0 server if they try to follow these instructions.

**Comment by ingenthr** [comment]
I thought @avsej said it's been there for a while in our call the other day.  Maybe do a quick search to see when it was added?

**Comment by amarantha-k** [comment]
@dnault Please let me know when we're ready to merge this PR.

**Comment by dnault** [comment]
@amarantha-k Ready. Thanks.

**Pull request merged**